var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -305,8 +328,23 @@ def configure(self):
                 if self.spec.satisfies("%clang") or self.spec.satisfies("%oneapi"):
                     config.write("enable_lto = False\n")
 
+    @run_after("install")
+    def copy_reference_images(self):
+        # https://matplotlib.org/devdocs/devel/testing.html#obtain-the-reference-images
+        install_tree(
+            join_path("lib", "matplotlib", "tests", "baseline_images"),
+            join_path(python_platlib, "matplotlib", "tests", "baseline_images"),
+        )
```

Comment:
IF you want to keep this you will want to also copy `lib/matplotlib/tests/*.pfb` files, probably more.

Reply:
Let's figure that out later if needed